lib/cloudflare_protected_email_address.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'cgi'
+
+class CloudflareProtectedEmailAddress
+  def initialize(obfuscated:)
```

Comment:
What's your thinking behind making this require a named parameter, rather than just a positional one? If there are likely to be other arguments that might make sense, but here it seems like it just complicates the interface.